use_cases/eluc/prescriptors/heuristics/heuristics.py

```diff
@@ -0,0 +1,127 @@
+"""
+Heuristic to compare our prescriptors to.
+"""
+from abc import ABC, abstractmethod
+
+import pandas as pd
+
+from data import constants
+from predictors.predictor import Predictor
+from prescriptors.prescriptor import Prescriptor
+
+class HeuristicPrescriptor(Prescriptor, ABC):
+    """
+    Abstract heuristic prescriptor class that inherits from prescriptor class.
+    Has a percentage threshold that the heuristic is to reach but not exceed.
+    Also takes a predictor so that we can evaluate metrics.
+    Requires an implementation of reco_heuristic which takes a context dataframe and returns
+    recommendations based on the heuristic.
+    """
+    def __init__(self, predictor: Predictor):
+        self.predictor = predictor
+
+    @abstractmethod
+    def _reco_heuristic(self, pct: float, context_df: pd.DataFrame) -> pd.DataFrame:
+        """
+        Abstract method that takes a percentage threshold of land change and a 
+        context dataframe and returns a dataframe of recommendations based on the heuristic.
+        """
+        raise NotImplementedError
+    
+    def prescribe_land_use(self, pct, __, context_df: pd.DataFrame) -> pd.DataFrame:
```

Comment:
Not entirely sure how to deal with this function header. I want prescribe_land_use to take in: parameter that modifies where prescription is along the pareto, context_df. Maybe I should redo how the trained prescriptors work by merging the id and results_dir parameters into a single tuple so this "_" isn't necessary.

Reply:
I think you can use the same parameters as for the super class:
```
self, cand_id: str, results_dir: Path, context_df: pd.DataFrame
```
Explain how you specifically use them in this implementation in the docstring:
Parameters:
    cand_id (str): The desired percentage of land use change
    results_dir (Path): Not used
etc.